spec/models/manageiq/providers/openstack/cloud_manager/refresh_spec_common.rb

```diff
@@ -863,5 +863,107 @@ def expect_sync_cloud_tenants_with_tenants_is_queued
       expect(sync_cloud_tenant.method_name).to eq("sync_cloud_tenants_with_tenants")
       expect(sync_cloud_tenant.state).to eq(MiqQueue::STATE_READY)
     end
+
+    def assert_targeted_vm(vm_name, attributes)
```

Comment:
I'm not using exactly the same, since it seemed like it was not possible to acquire the external network information from the information available from just the VM. It seemed like it would take basically a full pull of inventory from Neutron to get some of it, so I left the external network stuff out of the targeted refresh and dropped the external network checks from the test.